contents.json

```diff
@@ -17624,6 +17624,26 @@
       "date_added": "Apr 8 2019",
       "suggested_by": "@vpeschenkov",
       "stars": 203
+    },
+    {
+      "title": "GitHubSearch",
+      "category-ids": [
+        "github", 
+        "realm", 
+        "developer"
+      ],
+      "description": "GitHub client written using MVVM architecture with Router (without Rx and Interface Builder).",
```

Comment:
@Karambirov Hi! Remove (.) the period from the end of description, please.

Reply:
Done